lib/ui/painting.dart

```diff
@@ -3327,6 +3343,66 @@ class _GaussianBlurImageFilter implements ImageFilter {
   int get hashCode => hashValues(sigmaX, sigmaY);
 }
 
+class _DilateImageFilter implements ImageFilter {
+  _DilateImageFilter({ required this.radiusX, required this.radiusY });
+
+  final double radiusX;
+  final double radiusY;
+
+  // MakeBlurFilter
```

Comment:
remove comment

Reply:
Or correct it to the right name for consistency with the other _Filter objects. Personally I don't see the value in the comment, so not propagating that consistency makes sense as well.